edx-platform/learninghub-theme/lms/templates/instructor/edx_ace/enrollenrolled/email/body.html

```diff
@@ -1,95 +1,184 @@
+                        <img src="{% with_link_tracking banner_url %}" alt="Pearson LearningHub" style="
+                            display:block;
+                            height:auto;
+                            width:600px;
+                            padding:0px;
+                            text-align:center;">
+                    </a>
+                </td>
+            </tr>
+
+            <tr>
+                <td style="
+                    padding:20px 20px 10px;
+                    background-color: white;">
+                    <span style="font-size:14px">
+                        <span style="font-family:Arial,Helvetica,sans-serif">
+
+                            <span style="color:#333333">
+                                {% if full_name%}
+                                    Dear {{ full_name }},<br><br>
+                                {% else %}
+                                    Dear {{ email_address }},<br><br>
+                                {% endif %}
+                                You have been enrolled in {{ display_name }} at&nbsp;
+                            </span>
+
+                            <a href="{{ site_name }}" style="color:#007fa3;text-decoration:underline" title="{{ site_name }}" target="_blank" data-saferedirecturl="{{ site_name }}">
+                                {{ site_name }}
+                            </a>
+
+                            <span style="color:#333333">
+                                &nbsp;by a member of the course staff. The course should now appear on your&nbsp;
+                            </span>
+
+
+                            <a href="{{ site_name }}" rel="noreferrer" style="color:#007fa3;text-decoration:underline" title="" target="_blank" data-saferedirecturl="{{ site_name }}">
+                                {{ site_name }}
+                            </a>
+
+                            <span style="color:#333333">
+
+                                &nbsp;dashboard.
+                                <br><br>
+
+                                To start accessing course materials, please visit
+
+                                <a href="{% with_link_tracking dashboard_url %}" style="color:#007fa3;text-decoration:underline" title="{% with_link_tracking dashboard_url %}" target="_blank" data-saferedirecturl="{% with_link_tracking dashboard_url %}">
+                                    {{ site_name }}
+                                </a>
+
+                            </span><br><br>
+
+                            <span style="color:#333333">
+                                This email was automatically sent from&nbsp;
+                            </span>
+
+                            <a href="{{ site_name }}" style="color:#007fa3;text-decoration:underline" title="{{ site_name }}" target="_blank" data-saferedirecturl="{{ site_name }}">
```

Comment:
Please, add a space after the colon. @andrey-canon
Split the attributes on their own line to avoid line too long.